src/crate/testing/layer.py

```diff
@@ -201,8 +201,9 @@ def __init__(self,
                                         port or '4200-4299',
                                         transport_port or '4300-4399',
                                         settings)
-        start_cmd = (crate_exec, ) + tuple(["-C%s=%s" % opt
-                                            for opt in settings.items()])
+        # ES 5 cannot parse 'True'/'False' as booleans so convert to lowercase
+        start_cmd = (crate_exec, ) + tuple(["-C%s=%s" % ((key, str(value).lower()) if type(value) == bool else (key, value))
```

Comment:
`"%s"` already does `str(value)` so I think the same could also be written as `["-C%s=" % key + str(value).lower() for key, value in settings.item()]` without the explicit bool check